framework/widgets/LinkPager.php

```diff
@@ -74,6 +74,12 @@ class LinkPager extends Widget
      * @var string the CSS class for the disabled page buttons.
      */
     public $disabledPageCssClass = 'disabled';
+    
+    /**
+     * @var array An array with options to pass to the span tag which is generated inside the disabled list element.
+     */
+    public $disabledSpanOptions = [];
```

Comment:
Maybe it's better to rename this property to `disabledPaseElementOptions` to be consistent with `disabledPageCssClass` and `activePageCssClass` property?